cmd/cluster/stage/stage.go

```diff
@@ -64,6 +72,72 @@ func NewCmd() *cobra.Command {
 
 // RunCmd runs the "ocne node update" command
 func RunCmd(cmd *cobra.Command) error {
+	// If the cluster name is not set, check for an existing
```

Comment:
Do we want to take a CAPI Cluster name and namespace as well?  Using the graph, we can make a good guess of what the provider is and do the stage even if we don't have access to the cluster config.

Reply:
I think it is good to allow user to specify rather than force deduction, especially since we are taking about upgrade which has a major impact on the cluster.

Reply:
Agreed.  This code has been removed.  A cluster name or config file must be explicitly specified.